src/Algebra/Definitions.agda

```diff
@@ -26,6 +25,7 @@ module Algebra.Definitions
 open import Algebra.Core using (Op₁; Op₂)
 open import Data.Product.Base using (_×_; ∃-syntax)
 open import Data.Sum.Base using (_⊎_)
+open import Relation.Nullary.Negation.Core using (¬_)
```

Comment:
Nice, although I thought this might have been done in one of your earlier rounds of `import` cleaning up?